k8s/bib.yaml

```diff
@@ -104,24 +115,21 @@ spec:
         - name: datasets
           emptyDir:
             sizeLimit: 5Gi
-        - name: redis-data
-          emptyDir:
-            sizeLimit: 1Gi
         - name: tmp
           emptyDir:
             sizeLimit: 1Gi
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: service
+  name: bibxml
   labels:
-    app: service
+    app: bibxml
 spec:
   type: ClusterIP
   ports:
     - port: 80
-      targetPort: 8000
+      targetPort: http
```

Comment:
App is listening to port `8000`. https://github.com/ietf-tools/bibxml-service/blob/main/k8s/bib.yaml#L54